File: myblog-server/src/main/java/com/site/blog/model/entity/BlogCategory.java
package com.site.blog.model.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.extension.activerecord.Model;
import lombok.*;

import java.sql.Timestamp;
import java.time.LocalDateTime;

/**
 * @author yzq
 * @description
 * @date:Created time 2021/8/25 23:57
 * @modified By:
 */
@EqualsAndHashCode(callSuper = true)
@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class BlogCategory  extends Model<BlogCategory> {
    /**
     * 关系id
     */
    @TableId(type = IdType.ASSIGN_ID)
    private String relationId;
    /**
     * 博客id
     */
    private Long blogId;
    /**
     * 类别id
     */
    private Long categoryId;
    /**
     * 创建时间
     */
    private Timestamp createTime;
}

File: blog-sever/server-kt/src/main/kotlin/com/site/blog/model/entity/BlogCategory.kt
package com.site.blog.model.entity

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableId
import com.baomidou.mybatisplus.extension.activerecord.Model
import java.sql.Timestamp

/**
 * @author yzq
 * @description
 * @date:Created time 2021/8/25 23:57
 * @modified By:
 */
data class BlogCategory(
    /**
     * 关系id
     */
    @TableId(type = IdType.ASSIGN_ID)
    var relationId: String? = null,

    /**
     * 博客id
     */
    var blogId: String? = null,

    /**
     * 类别id
     */
    var categoryId: String? = null,

    /**
     * 创建时间
     */
    var createTime: Timestamp? = null,
) : Model<BlogCategory?>()